frontend/public/components/utils/dropdown.jsx

```diff
@@ -317,24 +324,79 @@ export class Dropdown extends DropdownMixin {
       if (_.has(headerBefore, key)) {
         rows.push(<li key={`${key}-header`}><div className="dropdown-menu__header" >{headerBefore[key]}</div></li>);
       }
-      rows.push(<DropDownRow className={klass} key={key} itemKey={key} content={content} onBookmark={storageKey && this.onBookmark} onclick={this.onClick} selected={selected} hover={hover} />);
+      rows.push(<DropDownRow className={klass} key={key} itemKey={key} content={content} onBookmark={storageKey && this.onBookmark} onclick={this.onClick} selected={selected} hover={hover} autocompleteFilter={autocompleteFilter} />);
     };
 
     _.each(items, (v, k) => addItem(k, v));
 
-    return <div className={classNames(className)} ref={this.dropdownElement} style={this.props.style}>
-      <div className={classNames('dropdown', dropDownClassName)}>
-        <button aria-haspopup="true" onClick={this.toggle} onKeyDown={this.onKeyDown} type="button" className={classNames('btn', 'btn-dropdown', 'dropdown-toggle', buttonClassName ? buttonClassName : 'btn-default')} id={this.props.id} aria-describedby={describedBy} >
-          <div className="btn-dropdown__content-wrap">
-            <span className="btn-dropdown__item">
+    //render PF4 dropdown markup if this is not the autcomplete filter
+    if(autocompleteFilter){
```

Comment:
I'm hesitant to keep two totally separate markup structures for these two dropdowns, but I don't have a great suggestion. Do we have a long term plan for avoiding this?

Reply:
we will have to see if we can covert the Project dropdown to something that resembles PF4. I have not attempted this yet, but I would probably save that one for last if possible...